src/scrapers/providers/universal/123movie.js

```diff
@@ -11,6 +11,9 @@ async function _123movie(req, sse){
     let season = req.query.season;
     let episode = req.query.episode;
 
+    // If it's a TV show, we need to strip the year
+    if(req.query.type === 'tv') queryTitle = queryTitle.substring(0, queryTitle.length - 7);
```

Comment:
Would this be a better approach to removing the year?
```suggestion
    if(req.query.type === 'tv') queryTitle = queryTitle.replace(/\s\([0-9]{4}\)$/, "");
```